src/04-modality-specific-files/06-physiological-and-other-continuous-recordings.md

```diff
@@ -97,9 +77,45 @@ sub-control01/
 {
    "SamplingFrequency": 100.0,
    "StartTime": -22.345,
-    "Columns": ["cardiac", "respiratory", "trigger"],
+   "Columns": ["cardiac", "respiratory", "trigger"],
    "cardiac": {
        "Units": "mV"
    }
 }
 ```
+
+## Additional rules
```

Comment:
Is it necessary to have a header here? If so, let's think of a more descriptive name.

Reply:
I think it helps. Everything prior to this header contains the "essential rules" ... everything below contains some half-complete recommendations on specific use cases.

Hence, how about `Recommendations for specific use cases`?